src/applications/vaos/services/appointment/index.js

```diff
@@ -81,15 +81,15 @@ function apptRequestSort(a, b) {
  * @param {Boolean} useV2CC Toggle fetching CC appointments via VAOS api services version 2
  * @returns {Appointment[]} A FHIR searchset of booked Appointment resources
  */
-export async function fetchAppointments({ startDate, endDate }) {
+export async function fetchAppointments({ startDate, endDate, avs = false }) {
```

Comment:
OOC will the `avs` param be true only when fetching past appointments? Or always? It doesn't need to block this PR, but if not it would be a nice enhancement to only call for past appointments, since an AVS can't exist for future appointments.

Reply:
@acrollet - From a FE perspective we're sending the flag for all booked appointments(future and past) but in vets-api, we have logic that only call the avs service if the booked appointment is past: https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/vaos/app/services/vaos/v2/appointments_service.rb#L436
We can add an enhancement to update this but the complexity would be slightly higher due to our `fetchAppointments` usage.

Reply:
oh perfect, I forgot about that check in vets-api. Thanks!